src/web_server/logics/excel_util.go

```diff
@@ -307,40 +308,52 @@ func buildAttrByPropertyType(rid, fieldName, cellValue string, field Property, r
 	return result, errMsg
 }
 
-func checkOrgnization(result map[string]interface{}, department map[int64]metadata.DepartmentItem,
+func checkOrgnization(result map[string]interface{}, department map[int64]metadata.DepartmentItem, rowIndex int,
 	defLang lang.DefaultCCLanguageIf, errMsg []string, fieldName, rid string) (map[string]interface{}, []string) {
 
 	if len(department) == 0 {
 		blog.Debug("no department in paas, rid: %s", rid)
-		errMsg = append(errMsg, defLang.Languagef("nonexistent_org"))
+		errMsg = append(errMsg, defLang.Languagef("web_excel_row_handle_error", fieldName, (rowIndex+1))+
+			defLang.Languagef("nonexistent_org"))
 		return result, errMsg
 	}
 	// convert Organization,  eg: "[1]总公司,[2]分公司" => "1,2"
 	orgStr := util.GetStrByInterface(result[fieldName])
 	if len(orgStr) <= 0 {
 		blog.Debug("get excel cell value failed, field:%s, value:%s, err:%v, rid: %s", fieldName,
 			result[fieldName], "not a valid organization type", rid)
-		errMsg = append(errMsg, "organization type is invalid")
+		errMsg = append(errMsg, defLang.Languagef("web_excel_row_handle_error", fieldName, (rowIndex+1))+
+			defLang.Languagef("nonexistent_org"))
 		return result, errMsg
 	}
 	orgItems := strings.Split(orgStr, ",")
 	org := make([]int64, len(orgItems))
 	for i, v := range orgItems {
 		var err error
 		orgID := orgBracketsRegexp.FindString(v)
+		if len(orgID) == 0 {
+			blog.Errorf("regular matching is empty, please enter the correct content, field:%s, value:%s, rid: %s",
```

Comment:
```suggestion
			blog.Errorf("regular matching is empty, please enter the correct content, field: %s, value: %s, rid: %s",
```
冒号后面要加一个空格的，下面的报错信息也是